docs/manual-language-docs/lsp-rust-analyzer.md

```diff
@@ -136,8 +136,8 @@ The following is an example configuration for using lsp-mode with a remote rust-
     :action-handlers (ht ("rust-analyzer.runSingle" #'lsp-rust--analyzer-run-single))
     :library-folders-fn (lambda (_workspace) lsp-rust-analyzer-library-directories)
     :after-open-fn (lambda ()
```

Comment:
this one is a bad merge, can you delete :after-open-fn + the lambda.

Reply:
Sounds good, done!